### PR TITLE
Coveralls

### PR DESCRIPTION
**Pull request by tsipinakis**
Send test coverage statistics to coveralls to provide a visual way to
monitor the progress of our testing suite.

### CODE REVIEW

**Comment by tsipinakis** [comment]
I have no clue why it segfaults with clang bug it doesn't seem to mind paradoxically.

**Comment by bebehei** [comment]
Oh, wait @tsipinakis. I got an email, when you started this PR. I'll see coming, that after this PR, even every force push triggers an email. I don't like this. I do this quite often. (not "quite often", actually "too much".). I see, that I'll spam you with this. Can these e-mails get switched off?

**Comment by tsipinakis** [comment]
If you mean the travis emails it was because I started a new branch (had to test this on the main repo) and travis is configured to send emails for every build status change and every time the build fails.

**Comment by tsipinakis** [comment]
Case in point: I just did a force push and you (hopefully) didn't get an email.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/dunst-project/dunst.svg?branch=master)](https://travis-ci.org/dunst-project/dunst)
+[![Build Status](https://travis-ci.org/dunst-project/dunst.svg?branch=master)](https://travis-ci.org/dunst-project/dunst) [![Coverage Status](https://coveralls.io/repos/github/dunst-project/dunst/badge.svg)](https://coveralls.io/github/dunst-project/dunst)
```
[the remark this answers is not in the document]
Comment (tsipinakis) [reply README.md]: Nice catch!

**Comment by bebehei** [comment]
I just don't want it to send mails on force push in a PR. The mails on master/for every PR are bearable.

AFAICS the mails do not get sent  on force push. So everything good! :+1:

**Comment by bebehei** [comment]
So, I've merged this now and I have to say, I'm convinced, this is a good feature. It's not spammy and also smoothly adds value to our tests. Thanks @tsipinakis

**Comment by bebehei** [comment]
@tsipinakis A small thing seen in #475. Why do we count the executed lines for `test/*.c`, too? I guess it should be implied, that these files have 100% coverage!?

**Pull request merged**